### DISCUSSION TITLE
笔记时间显示小时和分钟

### DISCUSSION THREAD

**Discussion by Sojurice** [Tokinx/Wing#115] [想法]
作者您好，我没有开启时间格式化，笔记时间显示为年月日。希望能在年月日之后加上小时和分钟，这样时间更精确。

再次感谢作者大大做这么好的主题，等付费了一定支持！

**Comment by Tokinx** [comment]
非常棒的建议，新版本已经推送，更新后即可查看到效果。

**Comment by Sojurice** [comment]
已更新，有效果了，非常棒！
